system/datatypes/festival.php

```diff
@@ -306,7 +306,7 @@ function showSignup($form = null) {
 		 * pay.
 		 */
 		function handleSignup($arg) {
-			global $hyphaUrl, $hyphaLanguage, $hyphaPage;
+			global $hyphaUrl, $hyphaContentLanguage, $hyphaPage;
```

Comment:
In addition to introducing RequestContext, this commit also introduces the split between UI language and content language. This was originally a separate commit, I think. It is probably too much work to split these now (since then the original loadLanguage would need to be re-introduced and modified), but this split should probably be mentioned in the commit message.